case-study-orchestral/index.html

```diff
@@ -0,0 +1,2567 @@
+        background: var(--wp--preset--gradient--pale-ocean) !important;
+      }
+      .has-electric-grass-gradient-background {
+        background: var(--wp--preset--gradient--electric-grass) !important;
+      }
+      .has-midnight-gradient-background {
+        background: var(--wp--preset--gradient--midnight) !important;
+      }
+      .has-small-font-size {
+        font-size: var(--wp--preset--font-size--small) !important;
+      }
+      .has-medium-font-size {
+        font-size: var(--wp--preset--font-size--medium) !important;
+      }
+      .has-large-font-size {
+        font-size: var(--wp--preset--font-size--large) !important;
+      }
+      .has-x-large-font-size {
+        font-size: var(--wp--preset--font-size--x-large) !important;
+      }
+      .fixit {
+        display: inline-block !important;
+      }
+    </style>
+    <link
+      rel="stylesheet"
+      id="ct_public_css-css"
+      href="https://orchestral.ai/wp-content/plugins/cleantalk-spam-protect/css/cleantalk-public.min.css?ver=5.182"
+      type="text/css"
+      media="all"
+    />
+    <link
+      rel="stylesheet"
+      id="oxygen-css"
+      href="https://orchestral.ai/wp-content/plugins/oxygen/component-framework/oxygen.css?ver=4.0.2"
+      type="text/css"
+      media="all"
+    />
+    <style id="wpgb-head-inline-css" type="text/css">
+      .wp-grid-builder:not(.wpgb-template),
+      .wpgb-facet {
+        opacity: 0.01;
+      }
+      .wpgb-facet fieldset {
+        margin: 0;
+        padding: 0;
+        border: none;
+        outline: none;
+        box-shadow: none;
+      }
+      .wpgb-facet fieldset:last-child {
+        margin-bottom: 40px;
+      }
+      .wpgb-facet fieldset legend.wpgb-sr-only {
+        height: 1px;
+        width: 1px;
+      }
+    </style>
+    <link
+      rel="stylesheet"
+      id="cmplz-general-css"
+      href="https://orchestral.ai/wp-content/plugins/complianz-gdpr-premium/assets/css/cookieblocker.min.css?ver=6.2.4"
+      type="text/css"
+      media="all"
+    />
+    <link
+      rel="stylesheet"
+      id="monsterinsights-vue-frontend-style-css"
+      href="https://orchestral.ai/wp-content/plugins/google-analytics-premium/pro/assets/vue/css/frontend.css?ver=8.7.0"
+      type="text/css"
+      media="all"
+    />
+    <link
+      rel="stylesheet"
+      id="rank-math-analytics-stats-css"
+      href="https://orchestral.ai/wp-content/plugins/seo-by-rank-math/includes/modules/analytics/assets/css/admin-bar.css?ver=1.0.94"
+      type="text/css"
+      media="all"
+    />
+    <link
+      rel="stylesheet"
+      id="ssa-styles-css"
+      href="https://orchestral.ai/wp-content/plugins/simply-schedule-appointments/assets/css/ssa-styles.css?ver=2.5.7.6"
+      type="text/css"
+      media="all"
+    />
+    <script
+      type="text/javascript"
+      src="https://orchestral.ai/wp-content/plugins/agency-base/includes/js/get-root-font-size.js?ver=2.1.6"
+      id="get-root-font-size-js"
+    ></script>
```

Comment:
These blocks and more below are coming from the WordPress plugins and are best to be removed.